deps/rabbit/src/rabbit_fifo_dlx_worker.erl

```diff
@@ -27,6 +27,7 @@
 -include_lib("rabbit_common/include/rabbit.hrl").
 % -include_lib("rabbit_common/include/rabbit_framing.hrl").
 
+-compile(nowarn_deprecated_callback).
```

Comment:
This one should be simple enough to convert to `format_status/1`, just replace the `state` in the return value, with the map we are currently creating.